dockerdata/startup.py

```diff
@@ -36,7 +36,7 @@
 elif product == 'dnsdist':
     args = ['--supervised', '--disable-syslog']
     apienvvar = 'DNSDIST_API_KEY'
-    apiconftemplate = """webserver("0.0.0.0:8083", '{{ apikey }}', '{{ apikey }}', {}, '0.0.0.0/0')
+    apiconftemplate = """webserver("0.0.0.0:8083")
```

Comment:
I think we still want to set the API key, password and ACL like it was done before, so:
```suggestion
    apiconftemplate = """webserver("0.0.0.0:8083")
    setWebserverConfig({password='{{ apikey }}', apiKey='{{ apikey }}', acl='0.0.0.0/0'})
```